libsql-sqlite3/src/pager.h

```diff
@@ -26,6 +26,8 @@
   #define SQLITE_DEFAULT_JOURNAL_SIZE_LIMIT -1
 #endif
 
+#include "sqliteInt.h"
```

Comment:
again, if you just need libsql_create_wal, let's forward declare it as an opaque type:
```
typedef struct libsql_create_wal libsql_create_wal;
```
, rather than pulling in the whole fat header. Alternatively, put `libsql_create_wal` in a separate small header and include that.

Reply:
I'll remove those imports, it's just that clang is spitting errors at me in about every file because of missing imports. It seems that this is non relevant for sqlite because of the amalgation, but this is hell of a mess